docs/release_notes.md

```diff
@@ -1,5 +1,113 @@
 #
 
+## 4.3.2
+
+Release Date: 2024-04-xx
+
+This release represents preparation for work on 5.0.0 and important bug fixes for the 4.3 series.
+
+Most significantly, this release includes the new GenQuery2 parser and new iCommand `iquery`.  For anyone already running the unreleased GenQuery2 API plugin, please note that it will halt and report a conflict with 4.3.2.  Please uninstall the separate package before upgrading to 4.3.2.
+
+Other notable inclusions are fixes for keyword combinations and bad inputs, a number of documentation additions, and a few new deprecation declarations.
+
+This release consists of [125 commits from 7 contributors](https://github.com/irods/irods/compare/4.3.1...4.3.2) and [closed 153 issues marked for 4.3.2](https://github.com/irods/irods/issues?q=milestone%3A4.3.2).
+
+The latest binary packages for CentOS7, AlmaLinux8, RockyLinux9, Ubuntu20, Ubuntu22, Debian11, and Debian12 are available at <https://packages.irods.org/>.
```

Comment:
The angle brackets feel intentional. Is that true? Do they cause something special to happen?

Reply:
Cool cool.